New version: SpinGlassTensors v1.2.0

JuliaRegistrator opened the pull request:
- Registering package: SpinGlassTensors
- Repository: https://github.com/euro-hpc-pl/SpinGlassTensors.jl
- Created by: @lpawela
- Version: v1.2.0
- Commit: b730c21c5b3eca07c482712f2c6ca76fd31ff0f0
- Reviewed by: @lpawela
- Reference: https://github.com/euro-hpc-pl/SpinGlassTensors.jl/commit/b730c21c5b3eca07c482712f2c6ca76fd31ff0f0#commitcomment-149589604
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd60db7596f18fb37e1e846e3976945209c2750524b199cb5db5394a5d9b817073e3cc203d5481c280e35ffd49ad54632bcf6d16efe7d217c4ed74cff507a5eb02de029209a9995212a41b76af53a0af497e90ae90f3335ac4488cfaaa7b720207ee3e1c382dfe37c9d0673f43342b788a13132c4d87c1ce24223e831ee8e3d6973cb6e0b0fb6d75451a11fe8c446cb2bb883b147dcafae7f92de5cf6c2d8b8f76799823959d970889e8c2564922a7320d3 -->

lpawela commented:
We only support Julia up to 1.10 currently. Can this be merge as is?

GunnarFarneback commented:
[noblock]

I'm not sure whether it helps with automerge but your Julia compat is wrong.
```
julia = "1.9, 1.10"
```
doesn't mean what you think, see https://pkgdocs.julialang.org/v1/compatibility/#Caret-specifiers.

Tip: The [PackageCompatUI](https://github.com/GunnarFarneback/PackageCompatUI.jl) package can help you get the compat you intend.

JuliaRegistrator committed:
New version: SpinGlassTensors v1.2.0

UUID: 7584fc6a-5a23-4eeb-8277-827aab0146ea
Repo: https://github.com/euro-hpc-pl/SpinGlassTensors.jl.git
Tree: 4b8d96ca1ab07a8b57ffe683b0b337c95c0b3d3c

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

lpawela commented:
[noblock]
@GunnarFarneback thanks, the package helps a lot.
